AlphaWallet/Browser/ViewControllers/BrowserViewController.swift

```diff
@@ -43,7 +43,7 @@ final class BrowserViewController: UIViewController {
 
     lazy var webView: WKWebView = {
         let webView = WKWebView(
-            frame: .zero,
+            frame: .init(x: 0, y: 0, width: 40, height: 40),
```

Comment:
@oa-s what does the `40` imply?

Reply:
there is could be any non zero value, 40 as its our min image size we using